src/main/java/com/flightstats/hub/app/InternalPropertiesResource.java

```diff
@@ -12,15 +12,14 @@
 import javax.ws.rs.core.Response;
 import java.util.Properties;
 import java.util.TreeSet;
-import java.util.stream.Stream;
 
+@Slf4j
 @SuppressWarnings("WeakerAccess")
 @Path("/internal/properties")
 public class InternalPropertiesResource {
     public static final String DESCRIPTION = "Get hub properties with links to other hubs in the cluster.";
-    private final static Logger logger = LoggerFactory.getLogger(InternalPropertiesResource.class);
-    private final static String redacted = "[REDACTED]";
-    private final Stream<String> matchers = Stream.of("app_key", "api_key", "password");
+    private final SecretsFilter secretsFilter = new SecretsFilter();
```

Comment:
I would make class name singular (SecretFilter). Also would inject this class using DI. 
Other options would be make method static since this method does have any state.

Reply:
+1 to injection. Makes it something you can mock on a unit test.

Reply:
injected.